IFC4x3/Properties/f/FaultMovementType_1HEf8LNPb5ZAU_EKoyIhJa/Documentation.md

```diff
@@ -0,0 +1 @@
+Contains a human readable summary of the type of movement (e.g. dip-slip, strike-slip) on the ShearDisplacementStructure. To report an identifier from a controlled vocabulary, movementType_uri shall be used.
```

Comment:
Missed "ShearDisplacementStructure" and "movementType_uri" that require a references.